### PR TITLE
Add specific aria labels per item

### LINKED ISSUE #143: Audit the "Updating" aria live message
As part of the a11y review, see #129 

Not sure the "updating" message ('Updating... please wait.') should be used for the button aria-label. Yes, currently, the plugin cards change the update button aria label but they use 'updatingLabel'     => __( 'Updating %s...' ) referencing the plugin name.

Instead, 'Updating... please wait.' should be used for `wp.a11y.speak()`. 

Seems only Themes updates dispatch the "updating" message to the live region. It should be done for all update types (core, themes, plugins, translations), on all screens (update-core.php and the other screens).

### PR DESCRIPTION
**Pull request by swissspidy**
Fixes #143.

### CODE REVIEW

**Comment by obenland** [comment]
Is `data-name` used for themes anywhere yet?

**Comment by swissspidy** [comment]
@obenland Not yet. I'm adding it to `wp.updates.updateTheme`. Work in progress 🙂

File: src/js/shiny-updates.js [hunk 1]
```diff
@@ -672,24 +672,26 @@
...
 				.addClass( 'updating-message' )
+				.attr( 'aria-label', wp.updates.l10n.updatingLabel.replace( '%s', $notice.data( 'name' ) ) )
 				.text( wp.updates.l10n.updating );
+
 			wp.a11y.speak( wp.updates.l10n.updatingMsg, 'polite' );
 
 			return wp.updates.ajax( 'update-theme', args );
+		}
 
-		} else if ( 'themes-network' === pagenow ) {
```
Comment (obenland) [inline src/js/shiny-updates.js]: No `else if`? :)

Comment (swissspidy) [inline src/js/shiny-updates.js]: Not needed because of the return above

Comment (obenland) [inline src/js/shiny-updates.js]: Ah, that will conflict with #164. If this is ready, feel free to get it in and I'll fix conflicts over there.

**Pull request merged**